### PR TITLE
Pass through -Wl options

### PR DESCRIPTION
**Pull request by isuruf**
Sending this as a separate PR as mentioned in https://github.com/swig/cccl/pull/4#issuecomment-495087448

### CODE REVIEW

File: cccl [hunk 1]
```diff
@@ -162,6 +162,14 @@ EOF
         esac
         ;;
 
+    -Wl,*)
+        IFS=',' read -ra linkopt2 <<< "$(echo $1 | sed 's/-Wl,//')"
```
Comment (wsfulton) [inline cccl]: Shouldn't IFS be reset back to it's previous empty value afterwards?

File: cccl [hunk 2]
```diff
@@ -162,6 +162,14 @@ EOF
...
+        for linkarg in ${linkopt2[@]}
+        do
```
Comment (wsfulton) [inline cccl]: Using the same for do style as used elsewhere would be preferable.

File: cccl (hunk 1 above)
Comment (isuruf) [reply cccl]: I think this sets the value for the `read` command only

Comment (JL2210) [reply cccl]: @wsfulton The environment variable is changed for this specific command only, as it's included on the same line as the command.

**Comment by isuruf** [comment]
Ready for a review again

**Pull request merged**